packages/nimble-components/src/patterns/error/template.ts

```diff
@@ -1,8 +1,14 @@
 import { html } from '@microsoft/fast-element';
 import type { ErrorPattern } from './types';
+import { overflow } from '../../utilities/directive/overflow';
 
 export const errorTextTemplate = html<ErrorPattern>`
-    <div class="error-text" title="${x => x.errorText}" aria-live="polite">
+    <div
+        class="error-text"
+        ${overflow('errorHasOverflow')}
+        title="${x => (x.errorHasOverflow && x.errorText ? x.errorText : null)}"
```

Comment:
Probably worthwhile to write unit tests for this behavior against one (not all) of the components that use this pattern. I'm pretty sure we have some existing title overflow behavior tests for labels, etc.